Traceback (most recent call last):
  File "./bayramov.py", line 29, in <module>
    bayramovResult = bayramovODE.solve(solver="interalg", ftol=0.05)
  File "/usr/local/epd/lib/python2.7/site-packages/FuncDesigner-0.36-py2.7.egg/FuncDesigner/ode.py", line 93, in solve
    r = prob.solve(solver, **kwargs)
  File "/usr/local/epd/lib/python2.7/site-packages/openopt-0.36-py2.7.egg/openopt/kernel/baseProblem.py", line 235, in solve
    return runProbSolver(self, *args, **kwargs)
  File "/usr/local/epd/lib/python2.7/site-packages/openopt-0.36-py2.7.egg/openopt/kernel/runProbSolver.py", line 240, in runProbSolver
    solver(p)
  File "/usr/local/epd/lib/python2.7/site-packages/openopt-0.36-py2.7.egg/openopt/solvers/UkrOpt/interalg_oo.py", line 275, in __solver__
    interalg_ODE_routine(p, self)
  File "/usr/local/epd/lib/python2.7/site-packages/openopt-0.36-py2.7.egg/openopt/solvers/UkrOpt/interalgODE.py", line 12, in interalg_ODE_routine
    assert len(f) == 1, 'multiple ODE equations are unimplemented for FuncDesigner yet'
AssertionError: multiple ODE equations are unimplemented for FuncDesigner yet
